tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -4754,7 +4754,7 @@ public function testTimeout(array $data): void
 
         $this->assertEquals($longtext['headers']['status-code'], 202);
 
-        for ($i = 0; $i < 10; $i++) {
+        for ($i = 0; $i < 1; $i++) {
```

Comment:
We need this as 10 for cloud tests, MySQL never times out with only 1

Reply:
Done!